.github/workflows/delete-branch-on-pr-close.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: Delete RC testing branch upon PR closure
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  delete-branch:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
I notice there's already v3 for this action. Maybe we could try to use the latest version?

Reply:
updated, thank you !